.taskcluster.yml

```diff
@@ -2,81 +2,155 @@ version: 1
+              description: >-
+                A subset of WPT's "${chunk[0]}" tests (chunk number ${chunk[1]}
+                of ${chunk[2]}), run in the ${browser.channel} release of
+                ${browser.name}.
+              owner: ${event.pusher.email}
+              source: ${event.repository.url}
+            payload:
+              image: jugglinmike/web-platform-tests:0.21
+              maxRunTime: 7200
+              artifacts:
+                public/results:
+                  path: /home/test/artifacts
+                  type: directory
+              command:
+                - /bin/bash
+                - --login
+                - -c
+                - set -ex;
+                  ~/start.sh
+                    ${event.repository.url}
+                    ${event.ref}
+                    ${event.after}
+                    ${browser.name}
+                    ${browser.channel};
+                  cd ~/web-platform-tests;
+                  ./tools/ci/taskcluster-run.py
+                    ${browser.name}
+                    --
+                    --channel=${browser.channel}
+                    --log-wptreport=../artifacts/wpt_report.json
+                    --no-fail-on-unexpected
+                    --test-type=${chunk[0]}
+                    --this-chunk=${chunk[1]}
+                    --total-chunks=${chunk[2]};
+    - $if: tasks_for == "github-pull-request"
+      then:
+        $map: [{name: firefox, channel: nightly}, {name: chrome, channel: dev}]
+        each(browser):
+          $map:
+            - name: wpt-${browser.name}-${browser.channel}-stability
+              description: >-
+                Verify that all tests affected by a pull request are stable
+                when executed in ${browser.name}.
+              extra_args: '--verify'
+            - name: wpt-${browser.name}-${browser.channel}-results
+              description: >-
+                Collect results for all tests affected by a pull request in
+                ${browser.name}.
+              extra_args: '--no-fail-on-unexpected --log-wptreport=../artifacts/wpt_report.json'
+          each(operation):
+            taskId: {$eval: 'as_slugid(operation.name)'}
+            taskGroupId: {$eval: 'as_slugid("task group")'}
+            created: {$fromNow: ''}
+            deadline: {$fromNow: '24 hours'}
+            provisionerId: aws-provisioner-v1
+            workerType:
+              $if: event.repository.full_name == 'web-platform-tests/wpt'
+              then:
+                wpt-docker-worker
+              else:
+                github-worker
+            metadata:
+              name: ${operation.name}
+              description: ${operation.description}
+              owner: ${event.pull_request.user.login}@users.noreply.github.com
+              source: ${event.repository.url}
+            payload:
+              image: jugglinmike/web-platform-tests:0.21
```

Comment:
A somewhat unrelated question: how can we create a public/shareable DockerHub account?

Reply:
I don't know, but [I think we all want that](https://github.com/web-platform-tests/wpt/pull/12657#discussion_r212583279)